TesseractCore.xcodeproj/project.pbxproj

```diff
@@ -434,6 +446,11 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(inherited)",
+					/Users/Alexander/Projects/tesseractcore/External/Madness/build/Debug,
+					"$(USER_LIBRARY_DIR)/Developer/Xcode/DerivedData/TesseractCore-byzvyskleahrxbehbkbhivcpdwdh/Build/Products/Debug",
+				);
```

Comment:
Xcode always does this. Could you delete `FRAMEWORK_SEARCH_PATHS` from the `TesseractCore` target’s build settings please? Also, do we need to explicitly link the tests against `Madness.framework` as well?